components/settings/settings.js

```diff
@@ -17,30 +17,7 @@ var module = angular.module('bedrock-idp.settings', []);
 module.controller(settings);
 
 module.run(function(brNavbarService, brSessionService, config) {
-  brNavbarService.registerMenu('brSettings', {
-    slug: '/settings',
-    icon: 'fa fa-wrench',
-    label: 'Settings',
-    pageTitle: 'Settings',
-    visible: false,
-    init: function(scope) {
-      var menu = this;
-      scope.$watch(function() {
-        return brSessionService.session;
-      }, function(session) {
-        if(session && session.identity) {
-          menu.visible = true;
-          menu.url = config.data.idp.identityBaseUri + '/' +
-            session.identity.sysSlug + menu.slug;
-        } else {
-          menu.visible = false;
-        }
-      }, true);
-      // TODO: should be done elsewhere once
-      // get latest session information
-      brSessionService.get();
-    }
-  });
+
 });
 
 return module.name;
```

Comment:
At some point we need to further refactor the `components` dir so that each component isn't its own angular module (this is unnecessary), but rather each is just part of the idp angular module. Then we can remove these `return module.name` lines.